elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -34,3 +34,53 @@ window.PLRTE = function (uuid, options) {
     );
   });
 };
+
+// Override default implementation of 'formula'
+
+var Embed = Quill.imports.parchment.Embed;
+
+class MathFormula extends Embed {
+  static create(value) {
+    const node = super.create(value);
+    if (typeof value === 'string') {
+      this.updateNode(node, value);
+    }
+    return node;
+  }
+
+  static updateNode(node, value) {
+    // Wait for the page to be fully loaded. Mathjax will not work unless the page is fully loaded.
+    if (document.readyState === 'complete') {
+      let html = MathJax.tex2chtml(value);
+      let formatted = html.innerHTML;
+      // Without trailing whitespace, cursor will not appear at end of text if LaTeX is at end
+      node.innerHTML = formatted + '&#8201;';
+      MathJax.typeset();
+      node.contentEditable = 'false';
+      node.setAttribute('data-value', value);
+    } else {
+      var n = node;
```

Comment:
Is this necessary, or can we just pass `node` directly to `updateNode`?

Reply:
Ah, that's a leftover.